components/admin/admin/settings.py

```diff
@@ -42,6 +46,7 @@
     'django.contrib.sessions',
     'django.contrib.messages',
     'django.contrib.staticfiles',
+    'django.contrib.sites',
```

Comment:
Needed for [django-allauth](https://django-allauth.readthedocs.io/en/latest/installation.html#django)